tests/test_bugzilla.py

```diff
@@ -84,28 +92,28 @@ def test_pass():
     result = testdir.runpytest(*BUGZILLA_ARGS)
     result.assert_outcomes(0, 0, 1)
 
-
+'''
 def test_new_bug_failing(testdir):
     testdir.makeconftest(CONFTEST)
     testdir.makepyfile("""
         import os
         import pytest
 
-        @pytest.mark.bugzilla('1')
+        @pytest.mark.bugzilla({'1': {}})
```

Comment:
I am not sure if we want this extension here. It should stay simple comply with current readme in my opinion.